### ISSUE TITLE
Getting DB script errors while executing the Metrics & UMA scripts against latest H2 DB version 2.1.210

### ISSUE THREAD

**Issue by ShanikaWickramasinghe** [wso2/product-is#13639]
**How to reproduce:**
Trying to do a migration from IS 5.11.0 Latest U2 WUM (WUM LEVEL 143) to IS 5.12.0 Beta2. While getting 5.11.0 GA H2 UMA , Metrics script executed getting below errors and therefore unable to try out the migration against H2 latest (2.1.210).

Get a external H2 database configured with docker . Ref https://hub.docker.com/r/oscarfonts/h2
```
I. docker pull oscarfonts/h2

II. docker run -d -p 1521:1521 -p 81:81 -v /path/to/local/data_dir:/opt/h2-data -e H2_OPTIONS=-ifNotExists --name=MyH2Instance oscarfonts/h2

III. docker logs -f MyH2Instance

IV. Access http://<ip>:81
Give a database name in the UI browser console for the H2 (eg:- test2. So a new database will be created as test2 when you click on connect

```
Get the 5.11.0 GA pack from website https://wso2.com/identity-and-access-management/download-thank-you/?type=binary and get the pack updated to Latest U2 (I updated for U2 level 143)
Execute the script dbscripts/identity/uma/H2.sql against the H2 server

Getting below error

```
Table "IDN_UMA_PT_RESOURCE" not found; SQL statement:
CREATE TABLE IF NOT EXISTS IDN_UMA_PT_RESOURCE_SCOPE (
  ID             INTEGER AUTO_INCREMENT NOT NULL,
  PT_RESOURCE_ID INTEGER                NOT NULL,
  PT_SCOPE_ID    INTEGER                NOT NULL,
  PRIMARY KEY (ID),
  FOREIGN KEY (PT_RESOURCE_ID) REFERENCES IDN_UMA_PT_RESOURCE (ID) ON DELETE CASCADE,
  FOREIGN KEY (PT_SCOPE_ID) REFERENCES IDN_UMA_RESOURCE_SCOPE (ID) ON DELETE CASCADE
) [42102-210] 4
2S02/42102 [(Help)](https://h2database.com/javadoc/org/h2/api/ErrorCode.html#c42102)
```
![Screenshot from 2022-05-08 12-44-41](https://user-images.githubusercontent.com/31848014/167285952-ad7002b9-f725-44f3-9c94-b20ead98a114.png)

dbscripts/metrics/H2.sql

```
Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS METRIC_GAUGE (\000d\000a    ID BIGINT AUTO_INCREMENT PRIMARY KEY,\000d\000a    SOURCE VARCHAR(255) NOT NULL,\000d\000a    TIMESTAMP BIGINT NOT NULL,\000d\000a    NAME VARCHAR(255) NOT NULL,\000d\000a    [*]VALUE VARCHAR(100) NOT NULL\000d\000a)"; expected "identifier"; SQL statement:
CREATE TABLE IF NOT EXISTS METRIC_GAUGE (
    ID BIGINT AUTO_INCREMENT PRIMARY KEY,
    SOURCE VARCHAR(255) NOT NULL,
    TIMESTAMP BIGINT NOT NULL,
    NAME VARCHAR(255) NOT NULL,
    VALUE VARCHAR(100) NOT NULL
) [42001-210] 42001/42001 [(Help)](https://h2database.com/javadoc/org/h2/api/ErrorCode.html#c42001)

```

![Screenshot from 2022-05-08 12-49-20](https://user-images.githubusercontent.com/31848014/167286099-119a378d-048a-4b2d-8f7f-11f0cb63c7b7.png)

**Environment**
IS 5.11.0 U2 WUM updated to level 143
H2 2.1.210

**Comment by ashendes** [comment]
This is happening because H2 1.4.xx was used in 5.11.0 GA, hence, those are not compatible with H2 2.1.210.
These tables are getting created without an issue in their compatible DB versions.
5.11.0 GA - H2 1.4.xx :heavy_check_mark:
5.11.0 U2 updated - H2 2.1.210 :heavy_check_mark:

**Comment by ashendes** [comment]
I tried to reproduce the error several times but could not encounter any errors with UMA tables. The metrics DB script issue exists and needs to be fixed in 5.11.0.

**Comment by ShanikaWickramasinghe** [comment]
Removing the migration label as the issue is related to a DB script failure coming from 5.11.0

**Comment by piyarathnalakmali** [comment]
Closing as the issue is fixed by merging the above PR for 5.11.0 and the public is already fixed.
